src/provider/ethplorer.ts

```diff
@@ -0,0 +1,34 @@
+import fetch from 'isomorphic-fetch';
+import { TokenInfo, Holder } from '../types';
+
+const ETHPLORER_API_URL =
+    process.env.REACT_APP_ETHPLORER_API_URL || 'https://api.ethplorer.io';
+const ETHPLORER_API_KEY = process.env.REACT_APP_ETHPLORER_API_KEY;
+const DXDAO_ADDRESS = '0xa1d65E8fB6e87b60FECCBc582F7f97804B725521';
```

Comment:
you can get the address from the config store, see the rest of the app how we get it.